blog/2025/09-30-okd-4.20-release-notes.md

```diff
@@ -0,0 +1,531 @@
+
+This summary highlights key user-facing changes, improvements, and new features in OpenShift Core Networking.
+
+**Kubernetes Version Updates:**
+
+* OpenShift SDN and Cluster Network Operator (CNO) have been **rebased to Kubernetes 1.33**.
+
+**User-Defined Networks (UDN) Enhancements:**
+
+* Introduces **Tech Preview support for assigning static IPs to VMs** on Layer 2 Cluster User Defined Networks (CUDNs) using IPAMClaims.
+* **API Changes**: The CUDN and UDN CRDs have been updated with **new API fields to support preconfigured IP addresses**.
+* **Feature Gate**: The `PreconfiguredUDNAddresses` feature gate has been introduced.
+* **IPAMClaims CRD Update**: The IPAMClaims CRD has been **updated to version v0.5.1-alpha**.
+* **Multus NAD Webhook Race Condition Fix**: A race condition has been resolved, preventing a circular dependency issue during installation.
+
+**OVN-Kubernetes Debugging Improvements:**
+
+* **Must-gather for UDN/CUDN CRs**: **Must-gather now collects UDN and CUDN Custom Resource Definitions (CRDs)**.
+* **Must-gather for RouteAdvertisements**: **RouteAdvertisements are now included** in must-gather output.
+
+**Local Gateway (LGW) Enhancements:**
+
+* **Layer 2 UDN Support**: Local Gateway mode now **supports Layer 2 UDNs**.
+* **LGW L2 e2e Tests**: **End-to-end tests have been added** for Local Gateway Layer 2 functionality.
+
+**BGP Support Enhancements:**
+
+* **Feature Gate Lifted**: The **feature gate for BGP support has been lifted in OpenShift 4.20, making it generally available**.
+* **VRF-Lite Tests**: Downstream tests for VRF-Lite functionality have been added.
+
+***
+
+## OpenShift Console
+
+This Console update delivers significant UI/UX enhancements and extends plugin capabilities.
+
+* New extension points (`console.action/resource-provider`) allow plugins to **modify actions for Subscriptions and Jobs**.
+* The **Infrastructure Overview page now supports direct region selection**.
+* Replication Controllers, Deployments, Volumes, and several plugins (Topology, Helm, Pipelines) have **modernized workflows using hooks**.
+* The Alertmanager, Log Viewer, and Role Binding forms now use **consistent PatternFly components**.
+* **Integrated log viewing is available**, including for serverless functions.
+* **Navigation improvements** include a new "Ecosystem" section and removal of the "Routes" section.
+* The **YAML editor gains several features** (copy/paste, fullscreen, sticky scroll, settings).
+* **Custom Resource details and list pages now dynamically display columns** based on CRD definitions.
+* **Image accessibility is improved with alt text**.
+* **External links have updated styling and a "new tab" icon**.
+* **OpenShift authentication now supports sessions**.
+* The **terminal upgrades to xterm.js v5.5.0**.
+* **OpenShift CLI binaries are available for RHEL8/9**.
+* The Console plugin API adds `getSegmentAnalytics`, and the console layout now uses the **PatternFly grid system**.
+
```

Comment:
The OperatorHub has been re-named to Software Catalog. Users now need to select a namespace before accessing the Software Catalog. (No idea why.)